packages/puppeteer-core/src/cdp/Page.ts

```diff
@@ -1232,6 +1237,35 @@ export class CdpPage extends Page {
   ): Promise<DeviceRequestPrompt> {
     return await this.mainFrame().waitForDevicePrompt(options);
   }
+
+  // Use browser's singleton implementation, as Bluetooth simulation is implemented on the
+  // browser level.
+  async #cdpBluetoothEmulationManager(): Promise<CdpBluetoothEmulationManager> {
```

Comment:
If its on the browser Puppeteer should implement it on it not the page.
This is confusing API for anyone using Puppeteer, as this will affect other pages, the way I understand it.